www/shared/utils/google-analytics.test.js

```diff
@@ -0,0 +1,81 @@
+import Router from 'next/router';
+
+const mockedRouter = {
+    ...Router,
+    replace: jest.fn(),
+    events: {
+        on: jest.fn(),
+        off: jest.fn(),
+    },
+};
+
+beforeEach(() => {
+    jest.resetModules();
+    jest.clearAllMocks();
+
+    process.env.GA_TRACKING_ID = '12345';
+});
+
+it('should do nothing if env var is not defined', () => {
```

Comment:
I think all this tests are associado with `trackRouteChanged`, therefore we should use a `describe`. Furthermore, we are missing tests for the exported `ga`: test that ga works with or without the env car